test/nn/encoder/test_stype_encoder.py

```diff
@@ -324,40 +333,6 @@ def test_timestamp_feature_encoder_with_nan(encoder_cls_kwargs):
     assert (~torch.isnan(x)).all()
 
 
-def test_text_embedded_encoder():
```

Comment:
This test is no longer necessary because `text_embedded` is converted to `embedding` during materialization.